docs/06-contributors/99-maintainers/050-operations.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Operations Overview
+id: operations
+keywords: [operations]
+---
+
+## Coverage
+
+"Wing" encompases several projects/sites.
+This document covers operations for the following:
+
+- https://docs.winglang.io (https://github.com/winglang/docsite)
+- Wing (https://github.com/winglang/wing)
```

Comment:
Separate between assets that are user facing (such as npm/docs/homepage) and contributor-facing (such as the repos)